packages/react-card/src/stories/CardScale.stories.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+
+import { Body, Caption } from '@fluentui/react-text';
+
+import { Button } from '@fluentui/react-button';
+import { MoreVerticalRegular } from '@fluentui/react-icons';
+import { Card, CardHeader } from '../index'; // codesandbox-dependency: @fluentui/react-card ^9.0.0-beta
+import { makeStyles, shorthands } from '@griffel/react';
+import { tokens } from '../../../react-theme/src/tokens';
```

Comment:
shouldn't this use public api ?
```suggestion
import { tokens } from '@fluentui/react-theme';
```